pages/about/principal.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Mount Sion CBS Secondary School</title>
+    
+    <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js">
+   <script src="//code.jquery.com/jquery-2.2.1.js"></script>
+
+</head>
+ <body>
+    
+<div class="ui container">
+    <img class="ui centered medium image" src="../../assets/images/logo.jpg">
+        
+<div class="stackable ui tabular menu">
+    <a class="item" href="../../index.html">
+        Home 
+    </a>
+        <div class="ui simple dropdown item">
+            About
+            <i class="dropdown icon"></i>
+            <div class="menu">
+                <div class="item" href="./pages/about/principal.html"> Principal's Welcome</div>
+                <div class="item" href="./pages/about/history.html"> History</div>
+                <div class="item" href="./pages/about/mission.html"> Mission Statement</div>
+                <div class="item" href="./pages/about/staff.html"> Staff</div>
+                <div class="item" href="./pages/about/policies.html"> School Policies</div>
+                <div class="item" href="./pages/about/testimonials.html"> Testimonials</div>
+                <div class="item" href="./pages/about/inspection.html"> Inspection Reports</div>
+            </div>
+        </div>
+    </a>
+    <a class="item" href="./pages/curriculum.html">
+        Curriculum
+    </a>
+    <a class="item" href="./pages/history.html">
```

Comment:
These roots have changed, so for the menu on pages in the`about` dir you will need to step back out of that dir to access the pages in the `pages` dir. 
Using the history.html as example, if you are in principal.html and want to get to history the link is `href="../history.html`